src/code/code_800EC960.c

```diff
@@ -3,91 +3,105 @@
+    PAGE_CHANNEL_INFO = 6, // unused
+    PAGE_INTERFACE_INFO = 7,
+    PAGE_SE_FLAG_SWAP = 8,
+    PAGE_BLOCK_CHANGE_BGM = 9,
+    PAGE_NATURAL_SOUND_CONTROL = 10, // unused
+    PAGE_OCARINA_TEST = 11,
+    PAGE_SE_PARAMETER_CHANGE = 12,
+    PAGE_SCROLL_PRINT = 13,
+    PAGE_FREE_AREA = 14,
+    PAGE_LIM = 15
+};
+
+#define SCROLL_PRINT_BUF_SIZE 25
 
 extern f32 D_8012F6B4[]; // from audio_synthesis
 
 // stick float vals
-extern u32 D_801305B0;
-extern u32 D_801305B4;
+extern f32 D_801305B0;
+extern s8 D_801305B4;
 extern s8 D_801305B8;
 extern s8 D_801305BC;
 extern s8 D_801305C0;
-extern f32 D_801305C4[2];
-extern u8 D_801305CC;
-extern u8 sPrevChargeLevel; // 801305E0
-extern f32 D_801305E4[4];
-extern f32 D_801305F4;
-extern u8 D_801305F8[8];
-extern u8 D_80130600;
-extern s8 D_80130604;
-extern s8 D_80130608;
-extern s8 D_8013060C;
-extern s8 D_80130610;
-extern s8 D_80130614;
-extern s8 D_80130618;
-extern u8 D_8013061C;
-extern f32 D_80130620;
-extern s8 D_80130624;
-extern u16 D_80130628;
-extern u8 D_8013062C;
-extern u8 D_80130630;
-extern u32 D_80130634;
-extern u32 D_80130638;
-extern u8 D_8013063C;
-extern u8 D_80130640;
-extern u8 D_80130644;
-extern u8 D_80130648;
-extern Vec3f* D_8013064C;
-extern f32 D_80130650;
-extern u8 D_80130654;
-extern u8 D_80130658[]; // <= 112
-extern s8 D_801306C8[20];
+extern f32 D_801305C4[2];            // = {-15.0f, -65.0f}
+extern u8 sAudioIncreasingTranspose; // = 0
+extern u8 gMorphaTransposeTable[16]; // = {0, 0, 0, 1, 1, 2, 4, 6, 8, 8, 8, 8, 8, 8, 8, 8}
+extern u8 sPrevChargeLevel;          // = 0
+extern f32 D_801305E4[4];            // = {1.0f, 1.12246f, 1.33484f, 1.33484f} = 2**({0, 2, 5, 5}/12)
+extern f32 D_801305F4;               // = 1.0f
+extern u8 D_801305F8[8];             // = {127, 80, 75, 73, 70, 68, 65, 60}
+extern u8 D_80130600;                // = 0
+extern s8 D_80130604;                // = 2
+extern s8 D_80130608;                // = 0
+extern s8 sAudioDemoActive;          // = 0
+extern s8 sSpecReverb;               // = 0
+extern s8 sAudioEnvReverb;           // = 0
+extern s8 sAudioCodeReverb;          // = 0
+extern u8 D_8013061C;                // = 0
+extern f32 sAudioEnemyDist;          // = 0.0f
+extern s8 sAudioEnemyVol;            // = 127
+extern u16 D_80130628;               // = 0xFFFF
+extern u8 D_8013062C;                // = 0
+extern u8 D_80130630;                // = 0
+extern u32 D_80130634;               // = 0
+extern u32 D_80130638;               // = 0
+extern u8 D_8013063C;                // = 0
+extern u8 D_80130640;                // = 0
+extern u8 D_80130644;                // = 0
+extern u8 D_80130648;                // = 0
+extern Vec3f* sSariaBgmPtr;          // = 0
+extern f32 D_80130650;               // = 2000.0f
+extern u8 D_80130654;                // = 0
+extern u8 D_80130658[];              // 0x6D <= size <= 0x70
+extern s8 sSpecReverbs[20];          // = {0, 0, 0, 0, 0, 0, 0, 40, 0, 15, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0}
```

Comment:
there's nothing "asset"y about this data, so it should be fine to write it out in C (and will need to be so once data is migrated, this is just in preparation of that). I'm sure the incbins were just because the work to identify data types wasn't done yet.